### PR TITLE
toast notification added

### PR DESCRIPTION
**Pull request by mihir2004**
## What type of PR is this? (check all applicable)
- [x] ✨ Feature
## Description
This PR adds a **Toast Notification Component** using Tailwind CSS, which provides an easy way to display alert messages to users in a web application. The component is designed to be highly customizable and reusable, featuring:

- **Positioning**: The notification appears at the bottom-right of the screen and automatically adjusts for multiple notifications.
- **Color and Styling**: The default styling includes a blue background and blue text for the notification, but it's easily customizable to fit different alert types (e.g., success, error, warning).
- **Dismiss Button**: The notification includes a "Dismiss" button for user interaction to manually close the toast.
- **Icons**: An alert icon is provided by default, making the component suitable for informative notifications.
  
### Features:
- Easily customizable colors, icons, and text for different alert types.
- Responsive and functional on both desktop and mobile devices.
- Can be adapted for additional functionality, like auto-dismiss.

## Mobile & Desktop Screenshots/Recordings (if there are UI changes)
Desktop Toast Notification

## Added to documentation?
- [x] 🙅 no documentation needed

### CODE REVIEW

**Comment by mihir2004** [comment]
@bobbyiliev

File: components/toast-notification-alert.html [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+        <svg
+            class="w-5 h-5 mr-3 text-blue-700"
+            fill="currentColor"
+            viewBox="0 0 20 20"
+            xmlns="http://www.w3.org/2000/svg"
+        >
+            <path
+                d="M10 1a9 8 0 100 18 9 9 0 000-18zm-.3 14.7a1 1 0 11-1.4-1.4L9.5 12H7a1 1 0 110-2h2.5l-.6-2.3a1 1 0 111.9-.5l1 3a1 1 0 01-.2 1L9.7 15.7z"
+            />
+        </svg>
```
Comment (bobbyiliev) [inline components/toast-notification-alert.html]: Thank you for your contribution! The SVG looks odd: 
image

Would you mind changing it?

Comment (mihir2004) [reply components/toast-notification-alert.html]: I have changed the svg to notification icon and pushed the changes to my branch, now do I need to raise another pull request ?

**Review by bobbyiliev — COMMENTED** [review]
The new SVG thjat you've used agian seems AI generated :)

I will go ahead and close this PR as AI contributions are not accepted.

image

**Comment by mihir2004** [comment]
Hi @bobbyiliev, this was not ai generated, if you could help me in What kind of svg do you expect I can make those changes. 
I am new to opensource 
If you could guide me I have ui elements made and would like to push them one by one. 
I had pushed this as a test to check how git/GitHub works

**Comment by bobbyiliev** [comment]
Hey!

You could use an open licensed SVG lib like this one here for example: 

> https://www.svgrepo.com/

**Comment by mihir2004** [comment]
Oh okay, can I do this change and make a pr again

**Pull request closed without merging**